### DISCUSSION TITLE
model-viewer on iOS devices?

### DISCUSSION THREAD

**Discussion by Swapnesh-dev** [google/model-viewer#4857] [Q&A] [answered]
How does model-viewer internally use the USDZ file provided via the ios-src attribute to display both WebGL and AR content on iOS devices?

**Comment by elalish** [comment, marked as answer]
It doesn't. A GLB is necessary as that is the only file model-viewer itself reads and that's what's shown in WebGL. The USDZ is only used for iOS AR QuickLook because that's the only file they support - it's a separate app that model-viewer simply hands off to.
